src/containers/TexterTodoList.jsx

```diff
@@ -118,7 +131,18 @@ class TexterTodoList extends React.Component {
       <Empty title="You have nothing to do!" icon={<CheckCircleIcon />} />
     );
 
-    return <div>{renderedTodos.length === 0 ? empty : renderedTodos}</div>;
+    return (
```

Comment:
Snackbar is a stub notification -- once we get this in, we could use maybe https://www.npmjs.com/package/react-browser-notifications to do browser notifications